src/graphql/generatedTypes.ts

```diff
@@ -608,9 +605,42 @@ export interface UpdateMyProfile_updateMyProfile_profile_primaryAddress {
    * The ID of the object.
    */
   readonly id: string;
+  readonly primary: boolean;
   readonly address: string;
   readonly postalCode: string;
   readonly city: string;
+  readonly countryCode: string;
+  readonly addressType: AddressType | null;
+}
+
+export interface UpdateMyProfile_updateMyProfile_profile_addresses_edges_node {
```

Comment:
The amount of different naming conventions this interface applies is ambitious to say atleast 😅 . These graphql types seems to be automatically generated? Could we configure the naming conventions somehow?

Reply:
Yes, there is codegen that fetches these monsters from graphQL server and UI cannot rename these. Typings.ts was added to make common "shortcuts".